homeassistant/components/ohme/coordinator.py

```diff
@@ -3,69 +3,72 @@
 from datetime import timedelta
 import logging
 
-from ohme import ApiException, AuthException, OhmeApiClient
+from ohme import ApiException, OhmeApiClient
 
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
 from homeassistant.core import HomeAssistant
-from homeassistant.exceptions import ConfigEntryError
 from homeassistant.helpers.update_coordinator import DataUpdateCoordinator, UpdateFailed
 
 from .const import DOMAIN
 
 _LOGGER = logging.getLogger(__name__)
 
-type OhmeConfigEntry = ConfigEntry[OhmeCoordinator]
+type OhmeConfigEntry = ConfigEntry[list[DataUpdateCoordinator]]
```

Comment:
I would consider making a dataclass for the coordinators, this way you can just access the individual coordinators without iterating the list

Reply:
Got it - so then from sensors.py how would I pick out the correct coordinator?

Reply:
Check withings, we do that there as well. 

```py
@dataclass(slots=True)
class WithingsData:
    """Dataclass to hold withings domain data."""

    client: WithingsClient
    measurement_coordinator: WithingsMeasurementDataUpdateCoordinator
    sleep_coordinator: WithingsSleepDataUpdateCoordinator
    bed_presence_coordinator: WithingsBedPresenceDataUpdateCoordinator
    goals_coordinator: WithingsGoalsDataUpdateCoordinator
```
So then I can access it with
`entry.runtime_data.measurement_coordinator`